From 021d2eb175b8e46d4690cd9e1243301ddd903bcc Mon Sep 17 00:00:00 2001
From: Daniel Park <dan.mcpark84@gmail.com>
Date: Tue, 26 Jun 2018 13:41:58 +0900
Subject: [PATCH 11/20] Adds classless static route option to DHCP options.

- Classless static route option is defined at RFC 3442 and widely used in OpenStack.

Change-Id: I66200faa5414b02dae325238e3bf1f0442efd0d8
---
 utils/misc/src/main/java/org/onlab/packet/DHCP.java | 1 +
 1 file changed, 1 insertion(+)

diff --git a/utils/misc/src/main/java/org/onlab/packet/DHCP.java b/utils/misc/src/main/java/org/onlab/packet/DHCP.java
index 9e6f38f4e0..a17f47ba02 100644
--- a/utils/misc/src/main/java/org/onlab/packet/DHCP.java
+++ b/utils/misc/src/main/java/org/onlab/packet/DHCP.java
@@ -79,6 +79,7 @@ public class DHCP extends BasePacket {
         OptionCode_DHCPServerIp((byte) 54), OptionCode_RequestedParameters((byte) 55),
         OptionCode_RenewalTime((byte) 58), OPtionCode_RebindingTime((byte) 59),
         OptionCode_ClientID((byte) 61), OptionCode_CircuitID((byte) 82),
+        OptionCode_Classless_Static_Route((byte) 121),
         OptionCode_END((byte) 255);
 
         protected byte value;
-- 
2.15.2 (Apple Git-101.1)

